homeworks/elena-savitskaya/homework-4/firstScript.js

```diff
@@ -0,0 +1,48 @@
+const array = [1, 50, 2, 5, 7, 10, 33, 22, 77];
+
+let moreTenArr = array.filter(function First(number) {
```

Comment:
```suggestion
let moreTenArr = array.filter(function(number) {
```
Don't need a name for function here. Name for function should start from lower case